tests/common.go

```diff
@@ -909,11 +909,13 @@ func ValidateContext(ctx *scheduler.Context, errChan ...*chan error) {
 
 		Step(stepLog, func() {
 			log.InfoD(stepLog)
-			err := Inst().S.WaitForRunning(ctx, timeout, defaultRetryInterval)
-			if err != nil {
-				PrintDescribeContext(ctx)
-				processError(err, errChan...)
-				return
+			if !ctx.SkipPodValidation {
```

Comment:
Why was this change done?

Reply:
in case of  there is any failed volume in restored ctx, we are setting SkipPodValidation to skip checking for running status of that pod.

This how i set it at testcase level.
if len(failedVolumes) > 0 {
	appCtx.SkipPodValidation = true
}